src/FunctionApps/python/GenerateCSVs/ecr.py

```diff
@@ -1,8 +1,96 @@
-from typing import List, Any
-from GenerateCSVs.patient import PATIENT_COLUMNS
+import logging
 
-ECR_COLUMNS = PATIENT_COLUMNS + []
+from typing import List
+from GenerateCSVs.patient import PATIENT_COLUMNS, parse_patient_resource
+from GenerateCSVs.elr import extract_loinc_lab
+from GenerateCSVs.vxu import parse_immunization_resource
 
+ECR_COLUMNS = (
+    PATIENT_COLUMNS
+    + [  # Immunization columns
+        "immunizationVaccineCode",
+        "immunizationVaccineDescription",
+        "immunizationOccurrenceDateTime",
+    ]
+    + [  # Observation columns
+        "observationLoincCode",
+        "observationResult",
+        "observationEffectiveDateTime",
+    ]
+)
```

Comment:
I was worried because the VXU/ELR columns are not specific enough for the broader scope of the ECR data set (e.g. loincCode).  There are a few other implications, but they are kind of a wash given this work is just for our demo/pilot.

@bamader what would you think about changing the column names in elr.py (and vxu.py) to be more specific and then implementing Dan's suggestion?

Reply:
I'll be honest I'm not seeing the issue with the VXU/ELR columns. I do think we should define the columns in the respective files and just import here, but Spencer, can you clarify where you think the lack of specificity is happening? If we need to change a couple column names it's not really a big deal, but since this is just a prototype one-off we should be sure the change does at least carry some value (for either us or readability purposes).

Reply:
Maybe I'm overthinking... My concern was, for example, there is a column called "loincCode" in the ELR CSV.  Loinc codes are used pretty generally in a bunch of different places, so in an ECR you might have many different ways a loinc code could be used (order type, document type, medication route, etc).  I was suggesting that we qualify it to observationLoincCode.

But maybe we can have the best of both worlds by using list comprehension.  I just pushed a change to do this.  Let me know what you think!